src/main/java/io/cdap/plugin/http/common/BaseHttpConfig.java

```diff
@@ -87,6 +93,24 @@ public abstract class BaseHttpConfig extends ReferencePluginConfig {
     @Macro
     protected String tokenUrl;
 
+    @Nullable
+    @Name(PROPERTY_PROXY_URL)
+    @Description("Proxy URL. Must contain a protocol, address and port.")
```

Comment:
update md files also for these new fields

Reply:
updated